policies/conflict-of-interest.md

```diff
@@ -3,74 +3,92 @@ status: draft
 ---
 # Conflict of Interest Policy
 
-This document establishes policies for identifying, disclosing, and managing conflicts of interest within the CF.
+This document outlines the Conflict of Interest Policy for the Commonhaus Foundation (CF), setting the framework for identifying, disclosing, and managing potential conflicts of interest to protect the Foundation’s interest when it is contemplating entering a [transaction](#definitions) that might benefit the private interest of a [covered individual](#scope-of-application).
 
-This policy applies to all individuals with decision-making authority within the CF, including CF Council (CFC) members and officers, and extends to project leaders and members when their interests may influence Foundation activities.
-
-- [Definition of Conflict of Interest](#definition-of-conflict-of-interest)
+- [Scope of Application](#scope-of-application)
+- [Definitions](#definitions)
   - [Examples](#examples)
+  - [Procedures for Determining Compensation](#procedures-for-determining-compensation)
 - [Disclosing Potential Conflicts of Interest](#disclosing-potential-conflicts-of-interest)
-  - [Immediate Disclosure](#immediate-disclosure)
-  - [Annual Disclosure](#annual-disclosure)
-- [Managing Conflicts](#managing-conflicts)
-- [Transparency and Documentation](#transparency-and-documentation)
-- [Prohibited Acts](#prohibited-acts)
-- [Guidelines for Gifts and Hospitality](#guidelines-for-gifts-and-hospitality)
-- [Training and Compliance](#training-and-compliance)
-- [Procedures for Determining Compensation](#procedures-for-determining-compensation)
+- [Determining Whether a Conflict of Interest Exists](#determining-whether-a-conflict-of-interest-exists)
+  - [Review by the CFC](#review-by-the-cfc)
+  - [Records of Review](#records-of-review)
+- [Violations](#violations)
 - [Review of Policy](#review-of-policy)
 
-## Definition of Conflict of Interest
+## Scope of Application
+
+This policy applies to all individuals within the CF who possess decision-making authority. This includes:
+
+- Members of the Commonhaus Foundation Council (CFC)
+- Members of the Extended Governance Committee (EGC)
+- Appointed officers of the CF
+- Project representatives
+- General members, when their personal or professional interests could potentially influence the decisions or activities of the Foundation.
+- Key employees[^1]
+
+[^1]: If and when the CF employs full-time staff, a key employee is an employee whose total annual compensation (including benefits) from the foundation and its affiliates is more than $150,000 and who (a) has responsibilities or influence over the foundation similar to that of officers or Councilors; or (b) manages a program that represents 10% or more of the activities, assets, income, or expenses of the foundation; or (c) has or shares authority to control 10% or more of the foundation capital expenditures, operating budget, or compensation for employees.
```

Comment:
This clause is identifying a "key employee" (someone that has been hired, rather than elected) that is covered under the policy.

This key employee can't participate in decisions for which they have a personal interest; they can not be present in discussions related to changes to their salary, for example.

Reply:
Sure, but my point was: why is their current salary a criteria?

Reply:
It is a qualifier to indicate a substantial/substantive interest. I can clarify the threshold with a lawyer. This example (https://www.thenopi.org/toolkit/conflict-of-interest-policy) uses this instead: 

> 2. Financial Interest	A person has a financial interest if the person has, directly or indirectly, through business, investment, or family:
> 
> a. An ownership or investment interest in any entity with which the Corporation has a transaction or arrangement,
> 
> b. A compensation arrangement with the Corporation or with any entity or individual with which the Corporation has a transaction or arrangement, or
> 
> c. A potential ownership or investment interest in, or compensation arrangement with, any entity or individual with which the Corporation is negotiating a transaction or arrangement.
> 
> Compensation includes direct and indirect remuneration as well as gifts or favors that are not insubstantial.

Some alternatives as well: 
- https://www.opensourcematters.org/organisation/directors/policies/299-conflict-of-interest-disclosure-policy.html
- https://foundation.rust-lang.org/policies/conflict-of-interest-policy/ -- "a conflict between a Covered Official’s obligations to Foundation and their personal, business, or other interests."  -- this one actually has some helpful language.. I could try bringing some of this in? WDYT?

Reply:
Well if this is about US law... I'm afraid my opinion doesn't matter much. I'd remove the salary threshold if possible, but otherwise any of those alternatives (including the existing wording) sounds fine.

Reply:
can drop that part. Not universally included. We may just have to define substantive later.. which is fine, as we don't have the case at all right now. ;)